app/services/foreman_ansible/variables_importer.rb

```diff
@@ -87,6 +87,10 @@ def create_new_variables(variables)
       end
     end
 
+    def override_variables(variables)
+      update_variables(variables).each { |var| var.update(:override => false) }
```

Comment:
You will need to move `memo` from l.131 to l.133 for this to work.

More importantly, I do not think we want to undo the overrides, because that will disable matchers users may have set up. Maybe https://projects.theforeman.org/issues/26485 could be a way forward?

Reply:
@xprazak2, this works I believe. It iterates over the updated results.
It really does, that's what it should do. It's called override, so it override the variable by fresh import :upside_down_face:
But as I'm saying in the post, I'm open to drop this concept :+1: 
As of the issue that's how update behaves now :) so I accidentally addressed the issue.

Reply:
I run into, it does not seem like my Rails server is convinced :slightly_smiling_face:

![import-override](https://user-images.githubusercontent.com/2664090/76605626-10dd2800-6511-11ea-8f00-e80059d3cdf5.png)